packages/contractkit/src/utils/tx-params-normalizer.ts

```diff
@@ -42,7 +42,7 @@ export class TxParamsNormalizer {
     }
 
     if (!isEmpty(txParams.gatewayFeeRecipient) && isEmpty(txParams.gatewayFee)) {
-      txParams.gatewayFee = DefaultGatewayFee.toString(16)
+      txParams.gatewayFee = '0x' + DefaultGatewayFee.toString(16)
```

Comment:
I know this is a short PR but can we use the `utils` functions?

Reply:
hahaha, yes